src/components/product-card-grid/style.module.css

```diff
@@ -107,26 +107,16 @@
 }
 
 .sectionBodyCardWrapper {
-  display: block;
   flex: 1 1 0;
   min-width: 120px;
   padding: 12px;
   width: min-content;
 }
 
 .sectionBodyCard {
-  composes: hds-surface-low from global;
```

Comment:
Love how much simpler this is from a "i wanna make a layout with cards" perspective 💯

Reply:
Agreed! It was satisfying deleting the code now handled by the card. 😊